PARTNERS.md

```diff
@@ -0,0 +1,49 @@
+# OADP Partner Information
+
+## Important Announcement: Version Support Changes
+Starting in 2024, OADP will implement a streamlined version support policy. Red Hat will support only one version of OADP per OpenShift version to ensure better stability and maintainability.
+
+## Version Mapping
+
+### Current and Planned Supported Versions
+| OpenShift Version | OADP Version | Velero Version |
+|-------------------|--------------|----------------|
+| 4.22             | 1.6          | v1.18          |
+| 4.21             | 1.6          | v1.18          |
+| 4.20             | 1.5          | v1.16          |
+| 4.19             | 1.5          | v1.16          |
+| 4.18             | 1.4          | v1.14          |
+| 4.17             | 1.4          | v1.14          |
+| 4.16             | 1.4          | v1.14          |
+| 4.15             | 1.3, 1.4     | v1.12, v1.14    |
+| 4.14             | 1.3, 1.4     | v1.12, v1.14   |
```

Comment:
there are specific patch versions that are considered good for each OCP version?

example I assume 1.4.0+ is ok to be used in 4.16, but for 4.18, do we need like 1.4.4+ for example?

Reply:
@mateusoliveira43 in general, patch versions are supported on the same OCP versions as the original minor version -- so 1.4.4 should be supported on the same versions as 1.4.1. A related question -- from a customer support point of view, do we only ever support the latest patch version, or "latest N" patch versions, or all patch versions? i.e. if 1.4.4 is out, do we still support 1.4.3? 1.4.0?